torchchat/model.py

```diff
@@ -442,8 +442,11 @@ def setup_caches(self, max_batch_size, max_seq_length):
         self.max_seq_length = max_seq_length
         self.max_batch_size = max_batch_size
         for b in self.layers.values():
-            b.attention.kv_cache = KVCache(
-                max_batch_size, max_seq_length, self.config.n_local_heads, head_dim
+            # Lower the setup_cache call to the attention module because tensor
```

Comment:
head_dim as defined on 440 is no longer used, let's remove it

Reply:
Good point, thanks!